src/app/features/videomania/index.js

```
@@ -70,13 +106,23 @@ const videoManiaSlice = createSlice({
-        state.isFetching = false;
+        state.isVideoFetching = false;
       })
       .addCase(getVideosByCategory.rejected, (state, action) => {
-        state.isFetching = false;
+        state.isVideoFetching = false;
+        state.error = action?.payload;
+      })
+      .addCase(getTopVideo.pending, (state) => {
+        state.isTopVideoFetching = true;
+      })
+      .addCase(getTopVideo.fulfilled, (state, action) => {
```

ESLint

[error] 'action' is defined but never used.

(no-unused-vars)